File: 프로그래머스/lv1/이선영/푸드 파이트 대회.java
import java.util.*;
class Solution {
    public String solution(int[] food) {

        StringBuilder food1 = new StringBuilder();
        
        for(int i=1; i<food.length; i++){
            int num = food[i]/2;
            System.out.println(num);
            for(int j=0; j<num; j++){
                food1.append(i);
            }
        }
        
        StringBuilder answer = new StringBuilder();
        answer.append(food1).append(0);
        String food2 = food1.reverse().toString();    
        answer.append(food2);
        
        return answer.toString();
    }
}

File: 프로그래머스/lv1/leehandsub/푸드 파이트 대회.kt
class Solution {
    fun solution(food: IntArray): String {
        var answer: String = "0"

        for (i in (food.indices).reversed()) {
            for (j in 0 until food[i] / 2) {
                answer += i.toString()
                answer = i.toString() + answer
            }
        }
        return answer
    }
}
